dcspy/logitech.py

```diff
@@ -180,7 +175,7 @@ def button_handle(self) -> None:
         """
         button = self.check_buttons()
         if button.value:
-            self._send_request(button)
+            self._send_request(button, key_down=KEY_DOWN)
```

Comment:
If the LCD button was pressed then send a key down event, as we are ignoring key up events for these if not a push button action.